sys/include/phydat.h

```diff
@@ -75,6 +75,7 @@ enum {
     /* generic values */
     UNIT_UNDEF,     /**< unit undefined */
     UNIT_NONE,      /**< data has no physical unit */
+    UNIT_CTS,       /**< counts */
```

Comment:
That's what I did at the beginning but to be consistent with what the sensor return, I thought it would make sense